scalasig/scalasig/src/main/scala/scala/meta/scalasig/Binaries.scala

```diff
@@ -59,4 +59,13 @@ object Binaries {
       }
     }
   }
+
+  def list(paths: Path*): List[Binary] = {
+    var ress = List.empty[Binary]
```

Comment:
If you expect this List to be large enough for performance to be a concern a ListBuffer would be the higher performance solution.
```
    val ress = ListBuffer.empty[Binary]

     apply(paths.toList) { res =>
      ress += res
    }
    ress.result
```